build.psm1

```diff
@@ -696,19 +690,12 @@ function New-PSOptions {
     }
 
     if (-not $Runtime) {
-        $RuntimeFromDotNet = dotnet --info | ForEach-Object {
+        $Runtime = dotnet --info | ForEach-Object {
             if ($_ -match "RID") {
                 $_ -split "\s+" | Select-Object -Last 1
```

Comment:
We have an Issue to make a single package for Linux too and seems the same problem too. So I believe we should use the same logic for both. 
If we continue to use `dotnet --info` we could move the logic in separate function for Windows and Linux.